public/lessons/javascript/the_command_line.md

```diff
@@ -109,13 +113,15 @@ This stands for "print working directory" or "present working directory" (not "p
 * for this class, we recommend:
   * create a `code` directory inside your home directory
   * create a new directory inside `code` for each lesson or project
+> WARNING: On some windows systems, Command Prompt will open to `C:\Windows\System32`
 
 # Listing Directory Contents
 
 * when you type `ls` ("list") it shows the contents of the current directory
-  * **On Windows** you might need to type `dir` instead
+  * **On Windows** you need to type `dir` instead
```

Comment:
The reason for saying "might" here is that if students have "git bash" installed (or any of a number of other unix emulation packages) then "ls" will work on Windows. We haven't been rigorous about asking students on Windows to use bash and cmder, but maybe we should start. @Dangeranger what do you think?